concent_api/core/tests/test_integration_force_payment.py

```diff
@@ -381,8 +379,8 @@ def test_provider_send_correct_force_payment_concent_should_accept(self):
             fields       = {
                 'recipient_type': message.concents.ForcePaymentCommitted.Actor.Provider,
                 'timestamp': parse_iso_date_to_timestamp("2018-02-05 12:00:20"),
-                'amount_pending': amount_pending,
-                'amount_paid':    amount_paid,
+                'amount_pending': 0,
```

Comment:
I would suggest extracting all those `0` and `10` to test class fields, i.e. `self.amount_pending` and `self.amount_paid`

Reply:
:+1: done